### PR TITLE
DOCKM-283: Update AsyncAPI component to 1.0.0-x releases

### PR DESCRIPTION
**Pull request by kaiszybiak**
## Hey, I just made a Pull Request!

https://sda-se.atlassian.net/browse/DOCKM-283

Bildschirmfoto 2021-11-16 um 18 43 40

Bildschirmfoto 2021-11-16 um 18 43 22

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by m4s7 — APPROVED** [review]

File: plugins/api-docs/src/components/AsyncApiDefinitionWidget/AsyncApiDefinitionWidget.tsx [hunk 1]
```diff
@@ -45,87 +96,56 @@ const useStyles = makeStyles(theme => ({
...
-    '& .asyncapi__server-security-flow': {
-      background: theme.palette.background.default,
-      border: 'none',
+    '& .text-gray-700': {
+      color: theme.palette.grey['100'],
     },
-    '& .asyncapi__server-security-flows-list a': {
-      color: theme.palette.primary.main,
+    '& .panel--right': {
+      background: color(theme.palette.navigation.background)
```
Comment (Fox32) [inline plugins/api-docs/src/components/AsyncApiDefinitionWidget/AsyncApiDefinitionWidget.tsx:136]: Eigentlich sollte es auch eine `darken()` function bei material ui react geben.

https://github.com/mui-org/material-ui/blob/512896973499adbbda057e7f3685d1b23cc02de9/docs/src/pages/guides/interoperability/StyledComponentsTheme.tsx#L2

File: plugins/api-docs/package.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
     "clean": "backstage-cli clean"
   },
   "dependencies": {
-    "@asyncapi/react-component": "^0.23.0",
+    "@asyncapi/react-component": "1.0.0-next.21",
```
Comment (Fox32) [inline plugins/api-docs/package.json:33]: Btw, do you really want to use a fixed version here?

File: plugins/api-docs/src/components/AsyncApiDefinitionWidget/AsyncApiDefinitionWidget.tsx (hunk 1 above)
Comment (kaiszybiak) [reply plugins/api-docs/src/components/AsyncApiDefinitionWidget/AsyncApiDefinitionWidget.tsx]: updated

File: plugins/api-docs/package.json (hunk 2 above)
Comment (kaiszybiak) [reply plugins/api-docs/package.json]: updated

**Pull request closed without merging**